tests/data/ImportsFlattened.sol.txt

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
+
+// File: @remapping_2_brownie/BrownieContract.sol
+
+pragma solidity ^0.8.4;
+
+contract BrownieContract {
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: @styleofbrownie/BrownieStyleDependency.sol
+
+pragma solidity ^0.8.4;
+
+contract BrownieStyleDependency {
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: @dependency_remapping/DependencyOfDependency.sol
+
+pragma solidity ^0.8.4;
+
+contract DependencyOfDependency {
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: @remapping/contracts/Dependency.sol
+
+pragma solidity ^0.8.4;
+
+
+struct DependencyStruct {
+    string name;
+    uint value;
+}
+
+contract Dependency {
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: @dependency_remapping/DependencyOfDependency.sol
+
+pragma solidity ^0.8.4;
+
+contract DependencyOfDependency {
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: @remapping_2/Dependency.sol
+
+pragma solidity ^0.8.4;
+
+
+struct DependencyStruct {
+    string name;
+    uint value;
+}
+
+contract Dependency {
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: CompilesOnce.sol
+
+pragma solidity >=0.8.0;
+
+struct MyStruct {
+    string name;
+    uint value;
+}
+
+contract CompilesOnce {
+    // This contract tests the scenario when we have a contract with
+    // a similar compiler version to more than one other contract's.
+    // This ensures we don't compile the same contract more than once.
+
+    function foo() pure public returns(bool) {
+        return true;
+    }
+}
+
+// File: ./././././././././././././././././././././././././././././././././././MissingPragma.sol
```

Comment:
i wonder if we should resolve relative imports for the comment string?

Reply:
I thought the approach of matching what's written in the source import statement was convenient because it makes for convenient searching

I don't think this extreme nesting will be common on real repos